core/config/model/chassis.go

```diff
@@ -93,13 +93,20 @@ type ConfigStruct struct {
 
 // ClientStruct client structure
 type ClientStruct struct {
-	ServerURI       string                 `yaml:"serverUri"`
-	TenantName      string                 `yaml:"tenantName"`
-	RefreshMode     int                    `yaml:"refreshMode"`
-	RefreshInterval int                    `yaml:"refreshInterval"`
-	RefreshPort     string                 `yaml:"refreshPort"`
-	Autodiscovery   bool                   `yaml:"autodiscovery"`
-	APIVersion      ConfigAPIVersionStruct `yaml:"api"`
+	Type              string                 `yaml:"type"`
+	ServerURI         string                 `yaml:"serverUri"`
+	TenantName        string                 `yaml:"tenantName"`
+	RefreshMode       int                    `yaml:"refreshMode"`
+	RefreshInterval   int                    `yaml:"refreshInterval"`
+	RefreshPort       string                 `yaml:"refreshPort"`
+	Autodiscovery     bool                   `yaml:"autodiscovery"`
+	APIVersion        ConfigAPIVersionStruct `yaml:"api"`
+	ApolloServiceName string                 `yaml:"serviceName"`
+	ApolloEnv         string                 `yaml:"env"`
```

Comment:
service name， env， app we already have

Reply:
servicename, app and env can be different in the Apollo Configuration Center, the micro-service name and service-name in Apollo need not be same